tools/stress-cluster/chaos/README.md

```diff
@@ -0,0 +1,201 @@
+azure-sdk-chaos is an environment and set of tools for performing long-running tests with the Azure SDK that involve OS and system-level faults.
+
+The chaos environment is an AKS cluster (Azure Kubernetes Service) with several open-source chaos tools installed on it, currently managed by @benbp.
+
+### Table of Contents
+
+* [Installation](#installation)
+* [Access](#access)
+* [Getting Started](#getting-started)
+* [Configuring faults](#configuring-faults)
+* [Running the example test with a network fault](#running-the-example-test-with-a-network-fault)
+
+Technologies used:
+
+1. [Azure AKS](https://docs.microsoft.com/en-us/azure/aks/)
+1. [Kubernetes](https://kubernetes.io/)
+1. [Chaos Mesh](https://chaos-mesh.org/)
+
+## Installation
+
+You will need the following tools to create and run tests:
+
+1. [Docker](https://docs.docker.com/get-docker/)
+1. [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+1. Azure CLI
+
+## Access
+
+To access the cluster, run the following:
+
+```
+az login
+az account set --subscription 'Azure SDK Developer Playground'
```

Comment:
This information is in a public repo, provided nothing sensitive is shared that is fine, but I am wondering if folks outside our org are able to play along at home with this or whether it would only be useful for folks inside the Azure SDK team itself?

Reply:
One of my next tasks is to move our cluster out of playground and into one of our main subscriptions, upon which I'll automate the whole buildout. After doing that, I can then document the buildout instructions/scripts here so that anyone following the readme can choose which step of the automation they want to jump in on (or with the right settings and idempotency, it will either provision or just connect).